docs/src/components/productTemplate/TemplateDemo.tsx

```diff
@@ -67,34 +67,34 @@ export default function TemplateDemo() {
   const TEMPLATES = {
     [DEMOS[0]]: [
       {
-        name: 'Flexy - React Material Dashboard (CRA + Next.js)',
+        name: 'Devias Kit Pro - Client & Admin Dashboard',
         src: {
-          light: '/static/branding/store-templates/template-light4.jpeg',
-          dark: '/static/branding/store-templates/template-dark4.jpeg',
+          light: '/static/branding/store-templates/template-4light.jpg',
+          dark: '/static/branding/store-templates/template-4dark.jpg',
         },
-        href: 'https://mui.com/store/items/flexy-react-admin-dashboard/',
+        href: 'https://mui.com/store/items/devias-kit-pro/',
       },
       {
         name: 'Minimal - Client & Admin Dashboard',
         src: {
-          light: '/static/branding/store-templates/template-light1.jpeg',
-          dark: '/static/branding/store-templates/template-dark1.jpeg',
+          light: '/static/branding/store-templates/template-1light.jpg',
+          dark: '/static/branding/store-templates/template-1dark.jpg',
```

Comment:
- `jpg` is used more frequently in the codebase, so replaced it.
- putting the number first makes it easier to work with the images